docs/en/ingest-management/overview.asciidoc

```diff
@@ -40,6 +40,27 @@ It can be started from any available x64 architecture {agent} artifact.
 
 For more information, see <<fleet-server>>.
 
+[discrete]
+[[package-registry-intro]]
+== {package-registry}
+
+The {package-registry} is an online package hosting service for the integrations
+available for selection using the {fleet} UI in {kib}.
+
+{kib} connects to the {package-registry} at `epr.elastic.co` using the Elastic Package Manager,
+downloads the latest integration package and stores its assets in {es}. This process
+requires an internet connection as integrations are updated and released periodically.
```

Comment:
```suggestion
downloads the latest integration package and stores its assets in {es}. This process
requires an internet connection because integrations are updated and released periodically.
```

Reply:
Thanks!